src/resources/api.router.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const router = require('express').Router();
+const controller = require('./api.controller');
+
+router.route('/todos')
```

Comment:
Mapping to controller's methods.

- GET /todos => [controller.getTodos](https://github.com/duyoji/creating_api_with_express_and_postgresql/pull/8/files#diff-8ee93ce05a5e607ffaa033acc783982cR15)
- POST /todos => [controller.postTodo](https://github.com/duyoji/creating_api_with_express_and_postgresql/pull/8/files#diff-8ee93ce05a5e607ffaa033acc783982cR19)
- PUT /todos => [controller.putTodo](https://github.com/duyoji/creating_api_with_express_and_postgresql/pull/8/files#diff-8ee93ce05a5e607ffaa033acc783982cR23)
- DELETE /todos => [controller.deleteTodo](https://github.com/duyoji/creating_api_with_express_and_postgresql/pull/8/files#diff-8ee93ce05a5e607ffaa033acc783982cR27)